test/src/org/exist/TestUtils.java

```diff
@@ -59,28 +60,58 @@ public static void cleanupDB() {
                 final Txn transaction = pool.getTransactionManager().beginTransaction()) {
 
                 // Remove all collections below the /db root, except /db/system
-                Collection root = broker.getOrCreateCollection(transaction, XmldbURI.ROOT_COLLECTION_URI);
-                assertNotNull(root);
-                for (Iterator<DocumentImpl> i = root.iterator(broker); i.hasNext(); ) {
-                    DocumentImpl doc = i.next();
-                    root.removeXMLResource(transaction, broker, doc.getURI().lastSegment());
-                }
-                broker.saveCollection(transaction, root);
-                for (Iterator<XmldbURI> i = root.collectionIterator(broker); i.hasNext(); ) {
-                    XmldbURI childName = i.next();
-                    if (childName.equals("system"))
-                        continue;
-                    Collection childColl = broker.getOrCreateCollection(transaction, XmldbURI.ROOT_COLLECTION_URI.append(childName));
-                    assertNotNull(childColl);
-                    broker.removeCollection(transaction, childColl);
+                Collection root = null;
+                try {
```

Comment:
suggestion: 

```java
try(Txn transaction = pool.getTransactionManager().beginTransaction())){
```

Reply:
It is already in the `try-with-resources`:

```java
 try(final DBBroker broker = pool.get(Optional.of(pool.getSecurityManager().getSystemSubject()));
                final Txn transaction = pool.getTransactionManager().beginTransaction()) {
```

Note the semicolon after the broker acquisition!